library/src/main/java/com/schibsted/spain/barista/internal/matcher/BackgroundMatcher.kt

```diff
@@ -44,9 +45,16 @@ class BackgroundMatcher private constructor(@DrawableRes private val expectedDra
       return false
     }
 
-    val viewBitmap = DrawableToBitmapConverter.getBitmap(view.background)
-    val expectedBitmap = DrawableToBitmapConverter.getBitmap(expectedDrawable)
-    return BitmapComparator.compare(viewBitmap, expectedBitmap)
+    if (expectedDrawable is ColorDrawable) {
+      val viewDrawable = view.background as ColorDrawable
+      return viewDrawable.color == expectedDrawable.color &&
+              viewDrawable.alpha == expectedDrawable.alpha &&
+              viewDrawable.opacity == expectedDrawable.opacity
```

Comment:
alpha != opacity? :S

Reply:
🤷 honestly, I started comparing the values that looked important. But don't know the difference between alpha and opacity.